lib/environ/src/vmoffsets.rs

```diff
@@ -236,224 +236,224 @@ impl VMOffsets {
 /// Offsets for `VMContext`.
 impl VMOffsets {
     /// The offset of the `signature_ids` array.
-    pub fn vmctx_signature_ids_begin(&self) -> u64 {
+    pub fn vmctx_signature_ids_begin(&self) -> u32 {
         0
     }
 
     /// The offset of the `tables` array.
     #[allow(clippy::erasing_op)]
-    pub fn vmctx_imported_functions_begin(&self) -> u64 {
+    pub fn vmctx_imported_functions_begin(&self) -> u32 {
         self.vmctx_signature_ids_begin()
-            + self.num_signature_ids * u64::from(self.size_of_vmshared_signature_index())
+            + self.num_signature_ids * u32::from(self.size_of_vmshared_signature_index())
```

Comment:
As `u32`s, these adds and multiplies here and below are now vulnerable to overflow as well. Would you mind converting them to checked_add/checked_mul? It's ok to use `unwrap()` here for now.